src/components/UI/Modal/Login/Login.js

```diff
@@ -43,7 +43,7 @@ class Login extends Component {
   }
 
   render() {
-    const { click, forgotPasswordClick, loading, error } = this.props;
+    const { click, ForgotPasswordClick, loading, error } = this.props;
```

Comment:
Any particular reason for the casing change?

Reply:
hmm... not sure. Probably just a mass search and replace error.